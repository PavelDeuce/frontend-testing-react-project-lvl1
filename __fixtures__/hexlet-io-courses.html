<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="utf-8" />
    <title>Курсы по программированию Хекслет</title>
    <link rel="stylesheet" href="index.css" />
  </head>
  <body>
    <h1>Node.js-программист</h1>
    <img src="img.png" alt="Иконка профессии Node.js-программист" />
    <script src="index.js"></script>
    <script src=""></script>
    <script src="https://google.com"></script>
  </body>
</html>
