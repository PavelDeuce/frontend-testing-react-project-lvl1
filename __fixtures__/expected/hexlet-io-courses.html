<!DOCTYPE html><html lang="ru"><head>
    <meta charset="utf-8">
    <title>Курсы по программированию Хекслет</title>
    <link rel="stylesheet" href="hexlet-io-courses_files/hexlet-io-index.css">
  </head>
  <body>
    <h1>Node.js-программист</h1>
    <img src="hexlet-io-courses_files/hexlet-io-img.png" alt="Иконка профессии Node.js-программист">
    <script src="hexlet-io-courses_files/hexlet-io-index.js"></script>
    <script src=""></script>
    <script src="https://google.com"></script>
  

</body></html>
